docs/program-setup/bare_metal/bare_metal.md

```diff
@@ -1,22 +1,231 @@
-| Support for Mbed Enabled development boards | Available | Available |
-| RTOS | Not available | Available |
-| Storage | Ecosystem libraries | Available |
-| Mbed TLS | Not available | Available |
-| Mbed PSA | Not available | Available |
-
-To begin using the Mbed OS bare metal profile from Mbed OS 2, please follow our [instructions](../tutorials/migrating-to-mbed-os-5.html).
+Bare metal is a profile of Mbed OS for ultraconstrained hardware. It represents a different way of working with Mbed OS: Instead of enabling all features by default, as the full profile does, bare metal mandates only the smallest set of APIs that development boards require - driver APIs, platform APIs and a subset of the RTOS APIs. All other supported APIs are optional - you can pull them in if you need the features they enable, but you'll probably leave most of them out - reducing your application's final size.
+
+Bare metal uses a subset of Mbed OS's RTOS APIs, without becoming a full RTOS. It's therefore suitable for applications that do not require complex thread management. This not only simplifies your application code, it also means you can use APIs that are not thread safe. Just as important, you can use the code-optimized versions of the C standard libraries, `microlib` and `newlib-nano`, which are much smaller than the thread safe equivalents the full profile requires.
+
+The bare metal profile is determined at build time - your working environment includes the full Mbed OS library, but only the bare metal APIs are compiled.
+
+## Features
+
+For a breakdown of supported APIs, please see [the full API list](../apis/index.html).
+
+The Mbed OS build tools - Mbed CLI, Mbed Online Compiler and Mbed Studio - all support working with the bare metal profile.
+
+<span class="notes">**Note:** Because bare metal uses some APIs that Mbed OS classifies as RTOS APIs, some class names that traditionally belong in RTOS programming are used in bare metal. For example, bare metal uses the class `ThisThread` despite not using threads.</span>
+
+<table>
+    <thead>
+        <tr>
+            <th colspan="2">Features</th>
+            <th>Support details</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan="6">Core</td>
+            <td>Drivers</td>
+            <td>Available; enabled by default <br>(Except the class `USBCDC_ECM`)</td>
+        </tr>
+        <tr>        
+            <td>Events</td>
+            <td>Available</td>
+        </tr>
+        <tr>        
+            <td>HAL</td>
+            <td>Available</td>
+        </tr>
+        <tr>        
+            <td>Platform</td>
+            <td>Available; enabled by default</td>
+        </tr>
+            <tr>        
+            <td>RTOS APIs</td>
+            <td>Semaphore, Mutex, EventFlags, ThisThread: Available; enabled by default.<br><br> All other classes not enabled.</td>
```

Comment:
Same comment about formatting. Since you show on the second page what is enabled by default I would remove that information here.
```suggestion
            <td>Semaphore, Mutex, EventFlags, ThisThread: Available; enabled by default.<br><br> All other APIs are not supported.</td>
```